styles2.css

```diff
@@ -22,6 +22,11 @@ body{
     
 }
 
+.logo h1 {
```

Comment:
## Style your logo
Please help styling logo with red color and try checking for the right font, as this will help improve the quality of looks of the website.

## Add Href
Please add an href to your logo that links to home, that will make it easier for users to navigate and is commonly used and good web practice.